test/system/helpers.bash

```diff
@@ -847,6 +847,21 @@ function _ensure_container_running() {
     die "Timed out waiting for container $1 to enter state running=$2"
 }
 
+# Return the config digest of an image in containers-storage.
+# The input can be a named reference, or an @imageID (including shorter imageID prefixes)
+# Historically, the image ID was a good indicator of “the same” image;
+# with zstd:chunked, the same image might have different IDs depending on whether
+# creating layers happened based on the TOC (and per-file operations) or the full layer tarball
+function image_config_digest() {
+    echo "image_config_digest $1" >&2
+    local config=$(skopeo inspect --raw --config "containers-storage:$1")
+    echo "===config\n$config\n===" >&2
```

Comment:
`\n` is not interpreted here.

Reply:
`echo -e`

Reply:
It turns out configs are almost never multi-line, so I went with a `[[[`/`]]]` bracketing instead.

(Maybe we don’t need these logs longer-term at all.)